spec/spec_helper.rb

```diff
@@ -1,59 +1,73 @@
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../config/environment', __dir__)
 require 'rspec/rails'
 require 'sidekiq/testing'
 require 'capybara/rails'
 require 'capybara/rspec'
 require 'capybara/email/rspec'
 require 'shoulda-matchers'
-Dir[Rails.root.join("spec/support/**/*.rb")].each {|f| require f}
-Dir[Rails.root.join("app/helpers/**/*.rb")].each {|f| require f}
-Dir[Rails.root.join("spec/shared_examples/*.rb")].each {|f| require f}
+Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
+Dir[Rails.root.join('app/helpers/**/*.rb')].each { |f| require f }
+Dir[Rails.root.join('spec/shared_examples/*.rb')].each { |f| require f }
+
+Capybara.server = :puma, { Silent: true }
+Capybara.register_driver :chrome_headless do |app|
+  options = ::Selenium::WebDriver::Chrome::Options.new
+
+  options.add_argument('--headless')
+  options.add_argument('--no-sandbox')
+  options.add_argument('--disable-gpu')
+
+  Capybara::Selenium::Driver.new(app, browser: :chrome, options: options)
+end
+
+Capybara.javascript_driver = :chrome_headless
 
-Capybara.javascript_driver = :selenium_chrome_headless
 Zonebie.set_random_timezone
 FakeWeb.allow_net_connect = false
 ActiveRecord::Migration.check_pending! if defined?(ActiveRecord::Migration)
 
 RSpec.configure do |config|
   config.expect_with :rspec do |c|
-    c.syntax = [:should, :expect]
+    c.syntax = %i[should expect]
   end
   config.mock_with :mocha
 
   config.fixture_path = "#{::Rails.root}/spec/fixtures"
   config.use_transactional_fixtures = false
   config.infer_base_class_for_anonymous_controllers = false
-  config.order = "random"
+  config.order = 'random'
   config.infer_spec_type_from_file_location!
 
-  config.include ApplicationHelper, :type => :feature
-  config.include Devise::Test::ControllerHelpers, :type => :controller
+  config.include ApplicationHelper, type: :feature
+  config.include Devise::Test::ControllerHelpers, type: :controller
   config.include FactoryGirl::Syntax::Methods
-  config.include Requests::JsonHelpers, :type => :controller
-  config.extend  Requests::ControllerMacros, :type => :controller
+  config.include Requests::JsonHelpers, type: :controller
+  config.extend  Requests::ControllerMacros, type: :controller
 
   config.before(:suite) do
     DatabaseCleaner.strategy = :transaction
   end
 
   config.before(:each) do |example|
-    if example.metadata[:disable_transaction]
-      DatabaseCleaner.strategy = :truncation
-    else
-      DatabaseCleaner.strategy = :transaction
-    end
+    DatabaseCleaner.strategy = if example.metadata[:disable_transaction]
+                                 :truncation
+                               else
+                                 :transaction
+                               end
     DatabaseCleaner.start
   end
 
   config.before(:each, type: :feature) do
     # Для капибарных тестов заполним редис курсами, чтобы не пытался стянуть их из инета
```

Comment:
Also please rewrite such Russian comments in English. The project is international, and English-speaking developers may use it.